examples/trials/mnist-annotation/mnist.py

```diff
@@ -180,11 +180,11 @@ def main(params):
     test_acc = 0.0
     with tf.Session() as sess:
         sess.run(tf.global_variables_initializer())
-        """@nni.variable(nni.choice(50, 250, 500), name=batch_num)"""
-        batch_num = params['batch_num']
-        for i in range(batch_num):
-            batch = mnist.train.next_batch(batch_num)
-            """@nni.variable(nni.choice(1, 5), name=dropout_rate)"""
+        """@nni.variable(nni.choice(1, 4, 8, 16, 32), name=batch_size)"""
+        batch_size = params['batch_size']
+        for i in range(params['batch_num']):
+            batch = mnist.train.next_batch(batch_size)
+            """@nni.variable(nni.choice(0.1, 0.5, 1), name=dropout_rate)"""
             dropout_rate = params['dropout_rate']
             mnist_network.train_step.run(feed_dict={mnist_network.images: batch[0],
                                                     mnist_network.labels: batch[1],
```

Comment:
mnist_network.keep_prob: dropout_rate
Literally, keep_prob should be 1-dropout_rate,  is this a mistake?

Reply:
thanks a lot, fixed, and also fixed the same problem in the other examples.